llama_stack/providers/tests/test_cases/chat_completion.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
do you feel we could use YAML instead of JSON? it lends more easily to commenting parts and is less "token-ey" to read